pkg/scheduler/extender_test.go

```diff
@@ -93,6 +93,7 @@ func TestSchedulerWithExtenders(t *testing.T) {
 			registerPlugins: []tf.RegisterPluginFunc{
 				tf.RegisterFilterPlugin("TrueFilter", tf.NewTrueFilterPlugin),
 				tf.RegisterQueueSortPlugin(queuesort.Name, queuesort.New),
+				tf.RegisterScorePlugin("EqualPrioritizerPlugin", tf.NewEqualPrioritizerPlugin(), 20),
```

Comment:
can we have a unit test without a prioritizer where we see that `EvaluatedNodes` is one?

Reply:
added a new testcase, PTAL